frontend/src/components/CoursePage/CoursePage.js

```diff
@@ -93,7 +93,7 @@ const CoursePage = (props) => {
     if (!auth.isAuthenticated()) {
       auth.login();
     }
-  });
+  }, []);
```

Comment:
Haven't looked over this too deeply but should this array contain auth.isAuthenticated()? May be a lot of computation, so there may be a better way to do it. Let me know what you think

Reply:
Hmm I'm not sure. Isauthenticated just checks localstorage so it's not big computation but I was thinking this code should only ever run once, idk. Either way we need to get rid of this code. The login popup should not come up immediately. We should show the blurred version of the site and have the user choose to login, instead of immediately bombarding them with the login

Reply:
I'm cool with merging this PR now and fixing this in another issue (maybe #3) if that's ok with you, in the interest of making this securely available to people who want to try it out.